src/main/java/edu/greenblitz/robotName/subsystems/swerve/Chassis/SwerveChassis.java

```diff
@@ -416,11 +416,13 @@ public boolean isAtPose(Pose2d goalPose) {
         boolean isAtX = Math.abs(goalPose.getX() - robotPose.getX()) <= TRANSLATION_TOLERANCE;
         boolean isAtY = Math.abs(goalPose.getY() - robotPose.getY()) <= TRANSLATION_TOLERANCE;
 
-        Rotation2d angDifference = (goalPose.getRotation().minus(robotPose.getRotation()));
-        boolean isAtAngle = angDifference.getRadians() <= ROTATION_TOLERANCE
-                || (Math.PI * 2) - angDifference.getRadians() <= ROTATION_TOLERANCE;
+        return isAtAngle(goalPose.getRotation()) && isAtX && isAtY;
+    }
 
-        return isAtAngle && isAtX && isAtY;
+    public boolean isAtAngle(Rotation2d goalAngle) {
+        Rotation2d angDifference = (goalAngle.minus(goalAngle));
```

Comment:
![image](https://github.com/GreenBlitz/CRRRRESCENDO/assets/26098399/491d2796-679e-4d7d-b41a-fa0aef8c59ad)

Water. Earth. Fire. Air.  
Long ago, the four nations lived together in harmony. Then, everything changed when the Fire Nation started writing unclear code. Only the Avatar, master of all four elements and clean code, could stop them, but when the world needed him most, he vanished. A hundred years passed and my brother and I discovered the new Avatar, an airbender named Ang, and although his clean code skills are great, he has a lot to learn before he's ready to save anyone. But I believe Ang can save this variable.